io.catenax.single_level_bom_as_built/1.0.0/SingleLevelBomAsBuilt.ttl

```diff
@@ -0,0 +1,164 @@
+   samm:description "The provided regular expression ensures that the UUID is composed of five groups of characters separated by hyphens, in the form 8-4-4-4-12 for a total of 36 characters (32 hexadecimal characters and 4 hyphens), optionally prefixed by \"urn:uuid:\" to make it an IRI."@en ;
+   samm:see <https://datatracker.ietf.org/doc/html/rfc4122> ;
+   samm:value "(^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)|(^urn:uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)" .
+
+:ChildData a samm:Entity ;
+   samm:preferredName "Child Data"@en ;
+   samm:description "Catena-X ID and meta data of the assembled child part."@en ;
+   samm:properties ( :createdOn :quantity [ samm:property :lastModifiedOn; samm:optional true ] :lifecycleContext :catenaXId [ samm:property :businessPartner; samm:optional true ] ) .
+
+:createdOn a samm:Property ;
+   samm:preferredName "Created On"@en ;
+   samm:description "Timestamp when the relation between the parent part and the child part was created, e.g. when the serialized child part was assembled into the given part."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime .
+
+:quantity a samm:Property ;
+   samm:preferredName "Quantity"@en ;
+   samm:description "Quantity of which the child part is assembled into the parent part. In general it is '1' for serialized parts."@en ;
+   samm:characteristic :QuantityCharacteristic .
+
+:lastModifiedOn a samm:Property ;
+   samm:preferredName "Last Modification Date"@en ;
+   samm:description "Timestamp when the assembly relationship between parent part and child part was last modified."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime .
+
+:lifecycleContext a samm:Property ;
+   samm:preferredName "Lifecycle Context"@en ;
+   samm:description "The lifecycle context in which the child part was assembled into the parent part."@en ;
+   samm:characteristic :LifecycleContextCharacteristic ;
+   samm:exampleValue "AsBuilt" .
+
+:businessPartner a samm:Property ;
+   samm:preferredName "Business Partner"@en ;
+   samm:description "The supplier of the given child item."@en ;
+   samm:characteristic :BpnTrait ;
+   samm:exampleValue "BPNL50096894aNXY" .
+
+:QuantityCharacteristic a samm-c:Quantifiable ;
+   samm:description "Describes the quantity in which the child part is assembled in the given parent object by providing a quantity value and the measurement unit in which the quantity is measured."@en ;
+   samm:dataType :Quantity .
+
+:LifecycleContextCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "Lifecycle Context"@en ;
+   samm:description "The possible values for lifecycle context as defined by STEP: AsRequired, AsDesigned, AsPlanned, AsBuilt, AsMaintained, AsRecycled"@en ;
+   samm:see <http://private.pdm-if.org/web/pdm-if/recommended-practices1> ;
```

Comment:
die URL scheint nicht zugreifbar zu sein

Reply:
after discussing with MS3 approvers, I removed the entire property.